ci/minikube-1.11-default.env

```diff
@@ -6,8 +6,6 @@ export PATH="$PWD/bin:$PATH"
 
 export MINIKUBE_ARGS="--vm-driver=none --kubernetes-version=v${KUBE_VERSION}"
 export Z2JH_HELM_ARGS="-f minikube-config.yaml"
-export DISABLE_TEST_NETPOL=1
```

Comment:
This was removed, but we have no place where this variable is read that is modified, so perhaps there is some update in this PR that is missing?

Reply:
Ah, this was why the test were failing also I guess.

```
    @pytest.mark.skipif(os.getenv('DISABLE_TEST_NETPOL') == '1',
                        reason="DISABLE_TEST_NETPOL set")
```

Reply:
Ah no, it should be fine that this test runs for all our CI tests, and this skipif statement can be removed rather.